docs/content/docs/concepts/database.mdx

```diff
@@ -523,44 +523,47 @@ res.user.lang // > "fr"
 
 ### ID Generation
 
-Better Auth uses `nanoId` by default to generate unique IDs for users, sessions, and other entities. If you want to customize how IDs are generated, you can pass your own function through the adapter or database configuration.
+Better Auth uses `nanoId` by default to generate unique IDs for users, sessions, and other entities. If you want to customize how IDs are generated, you can now configure this in the `advanced` section of the <Link href="/docs/reference/options">options</Link>.
 
 **Guidelines:**
 
 - IDs need to be in string format.
 - The ID should be unique for each entity type (like users, sessions, etc.).
 - Since the session ID acts as the token, it should be both unique and sufficiently long for security.
 
-**Example: Built In Kysley Adapter**
+**Example Usage Of Custom ID Generation**
 ```ts title="auth.ts" 
 import { betterAuth } from "better-auth";
 import { db } from "./db";
 
 export const auth = betterAuth({
    database: {
-      db: db,
+      db: db
+   },
+   advanced: {
       generateId: () => {
          // custom id generation logic
       }
    }
 })
 ```
 
-**Example: Prisma Adapter**
-```ts title="auth.ts"
+You can also enable automatic database ID generation by setting the `useAutoGeneratedId` option to `true` in your adapter configuration.
+
+**Example Usage Of Automatic Database IDs**
+```ts title="auth.ts" 
 import { betterAuth } from "better-auth";
+import { db } from "./db";
 
 export const auth = betterAuth({
-   database: prismaAdapter(prisma, {
-      provider: "sqlite",
-      generateId: () => {
-         // custom id generation logic
-      }
-   })
+   database: {
+      db: db,
+      useAutoGeneratedId: true
```

Comment:
why add a new option when you can simply use `generateId => false`, which does the same. or can also be achieved with `generateId() => undefined`. This introduce 2 ways of achieving the same thing.